pyface/ui/qt4/layout_widget.py

```diff
@@ -0,0 +1,133 @@
+# (C) Copyright 2005-2021 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+from enum import Enum
+
+from traits.api import provides
+
+from pyface.qt import QtGui
+from pyface.i_layout_item import DEFAULT_SIZE, SizePolicy
+from pyface.i_layout_widget import ILayoutWidget, MLayoutWidget
+from pyface.ui.qt4.widget import Widget
+
+
+#: Maximum widget size (some versions of PyQt don't export it)
+QWIDGETSIZE_MAX = getattr(QtGui, "QWIDGETSIZE_MAX", 1 << 24 - 1)
+
+
+class SizePolicies(Enum):
+    """ Qt values for size policies
+
+    Note that Qt has additional values that are not mapped to Pyface size
+    policies.
+    """
+    fixed = QtGui.QSizePolicy.Fixed
+    preferred = QtGui.QSizePolicy.Preferred
+    expand = QtGui.QSizePolicy.Expanding
+
+
+@provides(ILayoutWidget)
+class LayoutWidget(MLayoutWidget, Widget):
+    """ A widget which can participate as part of a layout. """
+
+    def _set_control_minimum_size(self, size):
```

Comment:
it's not clear from the `MLayoutWidget` documentation that `size` needs to be a tuple where the first and second elements are width and height - although that might be the default behavior for most/all toolkits.

Reply:
Not _too_ concerned about that as this API is only used by toolkit implementers.  End-users will use the `minimum_size` trait which is set to the equivalent to `Tuple(Int, Int)`.